backend/src/main/java/ro/utcn/danf/a1/command/CommandFactoryImplementation.java

```diff
@@ -0,0 +1,23 @@
+package ro.utcn.danf.a1.command;
+
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+import ro.utcn.danf.a1.service.PostManagementService;
+import ro.utcn.danf.a1.service.SiteUserDetailsService;
+import ro.utcn.danf.a1.service.TagManagementService;
+
+@Component
+@RequiredArgsConstructor
+public class CommandFactoryImplementation implements CommandFactory {
+    private final PostManagementService postManagementService;
+    private final SiteUserDetailsService siteUserDetailsService;
+    private final TagManagementService tagManagementService;
+
+    @Override
+    public Command getAskQuestionCommand() {
```

Comment:
Overall good implementation of the command pattern, with the small caveat that you don't have an Invoker class. Also, I expect that you have more than one type of command.